homeassistant/bootstrap.py

```diff
@@ -289,7 +289,7 @@ async def stop_hass(hass: core.HomeAssistant) -> None:
         # do a clean stop without knowing what is broken
         with contextlib.suppress(TimeoutError):
             async with hass.timeout.async_timeout(10):
-                await hass.async_stop()
+                await hass.async_stop(force=True)
```

Comment:
The fix is right, right now nothing is happening.

The current `async_stop` is also going to always ensure things stop, so we don't need to wrap it with our own timeout anymore. 

I would suggest we remove `stop_hass` completely and instead just call `hass.async_stop(force=True)` in every instance that called `stop_hass`.